jobs/release-microk8s-beta/Jenkinsfile

```diff
@@ -0,0 +1,46 @@
+@Library('juju-pipeline@master') _
+
+pipeline {
+    agent {
+        label 'runner'
+    }
+    /* XXX: Global $PATH setting doesn't translate properly in pipelines
+     https://stackoverflow.com/questions/43987005/jenkins-does-not-recognize-command-sh
+     */
+    environment {
+        PATH = '/snap/bin:/usr/sbin:/usr/bin:/sbin:/bin:/usr/local/bin'
+        CDKBOT_TOKEN = credentials('cdkbot')
+    }
+    parameters {
+        string(name: 'DRY_RUN', defaultValue: 'no', description: 'Do not release. Run all tests but do not release.')
+        string(name: 'ALWAYS_RELEASE', defaultValue: 'no', description: 'This script will release only on a new patch release. Set this to "yes" if we need to release to beta even if the patch release number is the same.')
+        string(name: 'TRACKS', defaultValue: '', description: 'The tracks you want to test and release. For example latest 1.10 1.11 1.12. Leave empty to iterate over all tracks.')
+        string(name: 'TESTS_BRANCH', defaultValue: '', description: 'You might want to test with a set of tests that are in your own branch. Here is where you set the branch name.')
+    }
+    options {
+        ansiColor('xterm')
+        timestamps()
+    }
+    stages {
+        stage('Setup') {
+            steps {
+                installTools()
+                sh "snapcraft login --with ${env.CDKBOT_TOKEN}"
```

Comment:
you can remove this setup step all together, everything is already setup for you.